scripts/provision.sh

```diff
@@ -73,17 +71,18 @@ install_dependencies() {
   echo "Installing gem packages..."
   bundle install --quiet
 
-  echo "Installing binaries: bosh-init, spiff, cf..."
+  echo "Installing binaries: bosh-init"
   if [ ! -x /usr/local/bin/bosh-init ]; then
     sudo wget -q $BOSH_INIT_URL -O /usr/local/bin/bosh-init
     sudo chmod +x /usr/local/bin/bosh-init
   fi
 
-  if [ ! -x /usr/local/bin/spiff ]; then
-    wget -q $SPIFF_URL -O spiff_linux_amd64.zip
-    sudo unzip -qo spiff_linux_amd64.zip -d /usr/local/bin
-    sudo chmod +x /usr/local/bin/spiff
-    rm spiff_linux_amd64.zip
```

Comment:
Spiff is still needed for the bosh deployment. For now, this script will need to install both spiff and spruce.